code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1298,3 +1298,97 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	M.cultslurring = min(M.cultslurring + 3, 3)
 	M.stuttering = min(M.stuttering + 3, 3)
 	..()
+
+/datum/reagent/consumable/ethanol/triple_sec
+	name = "Triple Sec"
+	id = "triple_sec"
+	description = "A sweet and vibrant orange liqueur."
+	color = "#ffcc66"
+	boozepwr = 30
+	taste_description = "a warm flowery orange taste which recalls the ocean air and summer wind of the caribbean"
+	glass_icon_state = "glass_orange"
+	glass_name = "Triple Sec"
+	glass_desc = "A glass of straight Triple Sec."
+
+/datum/reagent/consumable/ethanol/creme_de_menthe
+	name = "Creme de Menthe"
+	id = "creme_de_menthe"
+	description = "A minty liqueur excellent for refreshing, cool drinks."
+	color = "#00cc00"
+	boozepwr = 20
+	taste_description = "a minty, cool, and invigorating splash of cold streamwater"
```

Comment:
Okay, this is a LITTLE bit wordy for a taste description.

Reply:
its just fluff so it doesn't matter

Reply:
I mean, it does. Try to cut down the wordiness please.

Reply:
explain

Reply:
Most of the other alchohol tastes are pithier, with a few exceptions. These are all REALLY long.

Reply:
Agreed, base reagents shouldn't given sentence-length flavors.

Reply:
This isn't even the longest one lol

Personally I'm a fan of giving a bit more description to flavors beyond "tastes like egg and cream", or simply "tastes like cola"; This is an RPG after all.

Reply:
Honestly I was planning on running through all of the other beverages and foods at some point to add a little more fluff; service jobs tend to rely more heavily on roleplaying (perhaps botany excepted) than others, so by having people able to make Dwarf Fortress style absurd concoctions that truly express how mad the combination of flavors would be, I think there's at least a little more fun to be had. Similarly it adds in some new opportunities for masking poisons in taste descriptions; "you can taste unadulterated pain and fire with a hint of sweetness" could easily describe a chili... or cyanide, which, from what we know, has a strong burning sensation. I can see it's a contentious direction, though.

Reply:
I mean, there's a limited amount of "balance" in terms of flavours, because you don't want EVERY reagent to have a unique taste. Most medications and poisons just taste of bitterness, with some exceptions. All of the taste descriptions pretty much I ported from bay, with some modifications here and there. They're supposed to be used in compounds.

Right now reagents can't have multiple tastes (as in, hops = 3, bitterness = 1), they can only have a single one, which I think leads to more giant compound tastes.